cpu/stm32/kconfigs/Kconfig.clk

```diff
@@ -368,4 +373,80 @@ config CLOCK_APB2_DIV
     default 8 if CLOCK_APB2_DIV_8
     default 16 if CLOCK_APB2_DIV_16
 
+if CPU_FAM_F0 || CPU_FAM_F1 || CPU_FAM_F3 || CPU_FAM_G0 || CPU_FAM_G4 || CPU_FAM_L0 || CPU_FAM_L1 || CPU_FAM_L4 || CPU_FAM_WB
+config CLOCK_ENABLE_MCO
+    bool "Enable MCU Clock Output (MCO) on PA8"
+
+choice
+bool "MCO source"
+depends on CLOCK_ENABLE_MCO
+default CLOCK_MCO_USE_PLLCLK
+
+config CLOCK_MCO_USE_PLLCLK
+    bool "Use PLLCLK as MCO source"
+
+config CLOCK_MCO_USE_HSE
+    bool "Use HSE as MCO source"
+
+config CLOCK_MCO_USE_HSI
+    bool "Use HSI as MCO source"
+
+config CLOCK_MCO_USE_LSE
+    bool "Use LSE as MCO source" if CPU_FAM_G0 || CPU_FAM_G4 || CPU_FAM_L0 || CPU_FAM_L1 || CPU_FAM_L4 || CPU_FAM_WB
+
+config CLOCK_MCO_USE_LSI
+    bool "Use LSI as MCO source" if CPU_FAM_G0 || CPU_FAM_G4 || CPU_FAM_L0 || CPU_FAM_L1 || CPU_FAM_L4 || CPU_FAM_WB
+
+config CLOCK_MCO_USE_MSI
+    bool "Use MSI as MCO source" if CPU_FAM_L0 || CPU_FAM_L1 || CPU_FAM_L4 || CPU_FAM_WB
```

Comment:
I think in this case a `depends on` fits better

Reply:
Yes, it works the same. So I changed it.